packages/grafana-runtime/src/config.ts

```diff
@@ -65,6 +65,13 @@ export class GrafanaBootConfig implements GrafanaConfig {
   featureToggles: FeatureToggles = {};
   licenseInfo: LicenseInfo = {} as LicenseInfo;
   rendererAvailable = false;
+  dashboardPreviews: {
```

Comment:
This is the only part I don't like...  but I think we can merge and improve from there.

I am globally nervous about this growing list of options/settings that are not really necessary.  In this case, I think we only need to flag the front end if there are problems.  If things are working fine, this info is not necessary.